op-batcher/batcher/driver.go

```diff
@@ -114,8 +114,9 @@ type BatchSubmitter struct {
 	txpoolState       TxPoolState
 	txpoolBlockedBlob bool
 
-	state         *channelManager
-	prevCurrentL1 eth.L1BlockRef // cached CurrentL1 from the last syncStatus
+	channelMgrMutex sync.Mutex // guards channelMgr and prevCurrentL1
```

Comment:
a conventional shorter name would be 
```suggestion
	channelMgrMu sync.Mutex // guards channelMgr and prevCurrentL1
```
up to you

Reply:
I think I'll leave this as is for now so it matches other mutexes in the same struct.